feat: Enable storage map querying (#598)
0xPolygonMiden/miden-node: crates/store/README.md

The **Store** maintains the state of the chain. It serves as the "source of truth" for the chain - i.e., if it is not in
the store, the node does not consider it to be part of the chain.

> Incoming requests to the store are trusted because they are validated at the RPC level.
> 

Reviewer: RPC level could be ambiguous imo

```suggestion
Incoming requests to the store are trusted because they are validated in the RPC component.

```
